### DISCUSSION TITLE
Is it possible to use GitHub Actions with a GUI independenty from GitHub?

### DISCUSSION THREAD

**Discussion by marco-eckstein** [community/community#37738] [Actions] [answered]
I know there is the open source [Github Actions Runner](https://github.com/actions/runner) that can be self-hosted. However, [the docs](https://docs.github.com/en/actions/hosting-your-own-runners/monitoring-and-troubleshooting-self-hosted-runners#checking-the-status-of-a-self-hosted-runner) state:

> **Checking the status of a self-hosted runner**
>
> A self-hosted runner can be located in either your repository, organization, or enterprise account settings on GitHub.

Thus, I wonder whether it is possible to use GitHub Actions completely independent of GitHub, and if so, I wonder where the GUI is that shows the workflow runs etc.?

**Comment by airtower-luna** [comment, marked as answer]
The answer is "Sort of, kind of". GitHub doesn't provide anything to run Actions without their services, self-hosted runners need to be attached to GitHub to receive jobs to run.

But [there is `act`](https://github.com/nektos/act), a project that essentially re-implements the Actions controller, and lets you run workflows locally. However, so far not all parts of the API (workflow language) are implemented. And of course it doesn't have access to events and authentication from GitHub. Whether that works for you depends on what your workflows need.

**Reply by marco-eckstein** [reply]
Thanks. Does act have a GUI or other way to overview workflow history? From skimming the docs, it seems it doesn't.

**Reply by airtower-luna** [reply]
Yep, as far as I know it just runs workflows on command. There's nothing that could keep history or anything.
